resources/external_jointcal.yaml

```diff
@@ -0,0 +1,215 @@
+description: Butler Data Repository Export
+version: 1.0.1
+data:
+- type: dataset_type
+  name: jointcalPhotoCalibCatalog
+  dimensions:
+  - band
+  - instrument
+  - skymap
+  - physical_filter
+  - tract
+  - visit_system
+  - visit
+  storage_class: ExposureCatalog
+  is_calibration: false
```

Comment:
I don't know exactly what this parameter is used for (I couldn't find a good docstring for it. But if this doesn't qualify as a "calibration" then it makes me question the name/usefulness of this parameter.

Reply:
Ah, so the term "calibration" is overloaded in the stack (and always has been).  A "calibration file" on one hand is a bias, dark, flat, fringe, also camera (gen3), etc.  On the other hand, there are photometric and astrometric calibrations but these are a different use of the term "calibration".  I didn't come up with these names...